nsot/migrations/0034_populate_interface_name_slug.py

```diff
@@ -28,5 +16,5 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.RunPython(add_name_slug, remove_name_slug)
```

Comment:
A comment here as to why this got changed to a no-op would be handy so that way people don't have to go digging through git history to understand. Otherwise, it could be confusing with the name of the migration not matching the forward action.